### ISSUE TITLE
Exception: not licensed by user  (Error Code: 400)

### ISSUE THREAD

**Issue by RolandVanZoest** [Esri/arcgis-python-api#1151]
Same as (closed) issue 1009 (https://github.com/Esri/arcgis-python-api/issues/1009)

For our situation it appeared to be a 'special non-esri license' causing the problem.

We have this (free) app called "Admin Tools for ArcGIS Online"
It's property "itemId" is "c34019b0623041608df4d06970a7a96a"

By excluding this license in file "_license.py" , function "user_entitlement()"
by adding two lines after 445 :
```
  445:   item_id = self.properties['listing']['itemId']
  446:   if item_id == "c34019b0623041608df4d06970a7a96a" :
  447:        return { }
```
the problem in our situation was solved.

Another option would be to drop the application/license, we do not use it that much.
Or solve this bug of course  ;-)

**Comment by achapkowski** [comment]
@RolandVanZoest what version of the API are you using?

**Comment by RolandVanZoest** [comment]
Hi Andrew,
The version shipped with AG-Pro 2.8 being *arcgis 1.8.5*
Roland

Op ma 14 feb. 2022 om 14:58 schreef Andrew ***@***.***>:

**Comment by achapkowski** [comment]
@RolandVanZoest based on some 3rd party libraries, I was able to test and show in v2.0.1 (next release) this should be fixed.  I'm going to keep it on our board but close it here.  If you still have trouble, let us know post 2.0.1.

Thanks for reporting this.
